database/migrations/2017_04_27_141453_create_submissions_table.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSubmissionsTable extends Migration
```

Comment:
What is a submission? Can a voucher be submitted multiple times? Are they related?